mindone/schedulers/__init__.py

```diff
@@ -0,0 +1 @@
+from .ddim_schedule import DDIMScheduler
```

Comment:
Looks like `DDIMScheduler` is missing

Reply:
it's in mindone/schedulers/ddim_schedule.py  Line 68